applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_adjoint_static_solver.py

```diff
@@ -83,18 +87,18 @@ def AddDofs(self):
 
     def Initialize(self):
         """Perform initialization after adding nodal variables and dofs to the main model part. """
-        if self.response_function_settings["response_type"].GetString() == "adjoint_local_stress":
-            self.response_function = StructuralMechanicsApplication.AdjointLocalStressResponseFunction(self.main_model_part, self.response_function_settings)
-        elif self.response_function_settings["response_type"].GetString() == "adjoint_nodal_displacement":
-            self.response_function = StructuralMechanicsApplication.AdjointNodalDisplacementResponseFunction(self.main_model_part, self.response_function_settings)
-        elif self.response_function_settings["response_type"].GetString() == "adjoint_linear_strain_energy":
-            self.response_function = StructuralMechanicsApplication.AdjointLinearStrainEnergyResponseFunction(self.main_model_part, self.response_function_settings)
-        elif self.response_function_settings["response_type"].GetString() == "adjoint_nodal_reaction":
-            self.response_function = StructuralMechanicsApplication.AdjointNodalReactionResponseFunction(self.main_model_part, self.response_function_settings)
+        if self.settings["response_function_settings"]["response_type"].GetString() == "adjoint_local_stress":
```

Comment:
completely cosmetic (so ignore if you like) but i think it would be cleaner to read

~~~py
response_type = self.settings["response_function_settings"]["response_type"].GetString()
if response_type == "...":
   ...
elif response_type == "...":


~~~

Reply:
forwarding to @armingeiser and @MFusseder

Reply:
I will change it.